src/shared/views/SparqlQueryView.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { match } from 'react-router';
+import * as queryString from 'query-string';
+import { push } from 'connected-react-router';
+import { Menu, Dropdown, Icon } from 'antd';
+import { ViewList, View } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+
+import ViewStatisticsProgress from '../components/Views/ViewStatisticsProgress';
+import SparqlQueryContainer from '../containers/SparqlQuery';
+
+const SparqlQueryView: React.FunctionComponent<{
+  match: match<{ org: string; project: string; viewId: string }>;
+  location: Location;
+  goToOrg(orgLabel: string): void;
+  goToProject(orgLabel: string, projectLabel: string): void;
+  goToView(
+    orgLabel: string,
+    projectLabel: string,
+    viewID: string,
+    viewType: string[] | string
+  ): void;
+}> = ({ match, location, goToOrg, goToProject, goToView }): JSX.Element => {
+  const {
+    params: { org: orgLabel, project: projectLabel, viewId },
+  } = match;
+  const [{ _results: views }, setViews] = React.useState<ViewList>({
+    '@context': {},
+    total: 0,
+
+    // @ts-ignore TODO fix incorrect typing in SDK
+    // Should be _results not _result!
+    // https://github.com/BlueBrain/nexus/issues/753
+    _results: [],
+  });
+  const nexus = useNexusContext();
+  const decodedViewId = decodeURIComponent(viewId);
+  // const targetView = views.find((view: View) => view['@id'] === decodedViewId);
+  const query = queryString.parse(location.search).query;
+
+  React.useEffect(() => {
+    nexus.View.list(orgLabel, projectLabel)
+      .then(setViews)
+      .catch(() => {
+        // 503 ?
```

Comment:
can we like, display an error message in the menu (as a disabled item) saying something went wrong while fetching the list of views?